code/modules/power/apc/apc_main.dm

```diff
@@ -109,6 +109,14 @@
 	var/datum/alarm_handler/alarm_manager
 	/// Offsets the object by APC_PIXEL_OFFSET (defined in apc_defines.dm) pixels in the direction we want it placed in. This allows the APC to be embedded in a wall, yet still inside an area (like mapping).
 	var/offset_old
+	var/cuttedAIwire = FALSE
+	var/unlocked = FALSE
+	var/syndicate_access = FALSE
+	var/away_general_access = FALSE
+	var/cell_5k = FALSE
+	var/cell_10k = FALSE
+	var/no_charge = FALSE
+	var/full_charge = FALSE
```

Comment:
you gotta DMDoc all of these man https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md